crates/humanode-runtime/src/frontier_precompiles.rs

```diff
@@ -51,6 +55,11 @@ where
             // Humanode precompiles:
             a if a == hash(2048) => Some(Bioauth::<R>::execute(handle)),
             a if a == hash(2049) => Some(EvmAccountsMapping::<R>::execute(handle)),
+            a if a == hash(2050) => {
```

Comment:
Why `2050`?

Reply:
`EvmAccountsMapping` has 2049.  2050 is `+1` as next evm related precompile.

Reply:
Can/should we use the same value at they use at moonbeam?

Reply:
It's the same value as well :)